src/features/referenceContextWatcher.ts

```diff
@@ -1,9 +1,11 @@
 import { window, workspace, commands, ExtensionContext } from 'vscode';
 
-import { getRefUriUnderCursor } from '../utils';
+import { getRefUriUnderCursor, getRefUnderCursor, getReferenceAtPosition } from '../utils';
```

Comment:
small thing: `getReferenceAtPosition` is not used

Reply:
thx! fixed now. didn't realize I had eslint off previously.